skyrim-platform/src/platform_lib/ThreadPoolWrapper.h

```diff
@@ -1,23 +1,25 @@
 #pragma once
+#include <cstdint>
 #include <ctpl/ctpl_stl.h>
 #include <memory>
 
 class ThreadPoolWrapper
 {
 public:
-  ThreadPoolWrapper(int numThreads_ = 1)
+  ThreadPoolWrapper(int32_t numThreads_ = 1)
     : numThreads(numThreads_)
   {
   }
 
-  auto Push(std::function<void(int)> f)
+  auto Push(std::function<void(int32_t)> f)
   {
-    if (!pool)
-      pool.reset(new ctpl::thread_pool(numThreads));
+    if (!pool) {
+      pool = std::make_unique<ctpl::thread_pool>(numThreads);
+    }
     return pool->push(f);
   }
 
 private:
   std::unique_ptr<ctpl::thread_pool> pool;
-  const int numThreads;
+  const int32_t numThreads;
```

Comment:
🤔

Reply:
See https://github.com/skyrim-multiplayer/skymp/blob/main/docs/docs_cpp_code_guidelines.md#types
If you think `int` is better you can propose changes of this document